Makefile

```diff
@@ -385,6 +392,20 @@ test-e2e-local-ha-multi-zone: $(GINKGO)
 test-e2e-local-operator: $(GINKGO)
 	./hack/test-e2e-local.sh operator --procs=$(PARALLEL_E2E_TESTS) --label-filter="default" ./test/e2e/operator/...
 
+test-pre-upgrade: $(GINKGO)
+	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter="pre-upgrade" ./test/e2e/gardener/...
+test-pre-upgrade-ha-single-zone: $(GINKGO)
+	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter="pre-upgrade" ./test/e2e/gardener/...
+test-pre-upgrade-ha-multi-zone: $(GINKGO)
+	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter="pre-upgrade" ./test/e2e/gardener/...
```

Comment:
Are we missing some additional labels here? This is always the same rule. Why do we need separate make targets then? 😕 
Same below.